readme.md

```diff
@@ -0,0 +1,184 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Math-magicians <a name="about-project"></a>
+
+**Todo-react** is an todos app. It contains three navigations items. It can store and retrieve your daily set of tasks from the browser's localstorage. It consist of three navigation items.
+<ul>
+<li>Home</li>
+<li>About</li>
+<li>Contact</li>
+</ul>
+ 
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">index.html</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">style.css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Javascript</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">script.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>ReactJS</summary>
+  <ul>
+    <li><a href="https://legacy.reactjs.org/docs/getting-started.html">ReactJS</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+*tasks won'be deleted even after refreshing the browser*<br>
+*Beautiful UI*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://iamsjunaid.github.io/todo-list-react) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: To clone or fork and run it in a browser
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone
+  https://github.com/iamsjunaid/todo-list-react.git
+```
+
+using Ubuntu:
+
+```sh
+  cd my-desired-folder
+  git clone 
+  https://github.com/iamsjunaid/todo-list-react.git
+```
+
+For more information on how to clone or fork a repository:
+- <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository">How to clone a repo</a>
+- <a href="https://docs.github.com/en/get-started/quickstart/fork-a-repo">How to fork a repo</a>
+
+### Install
+
+There is no installation required.
+
```

Comment:
## optional

kindly edit the Installation required for the users to have a better experience while installing this app